pkg/api/v1/types.go

```diff
@@ -639,6 +639,8 @@ type Container struct {
 	ImagePullPolicy PullPolicy `json:"imagePullPolicy" description:"image pull policy; one of PullAlways, PullNever, PullIfNotPresent; defaults to PullAlways if :latest tag is specified, or PullIfNotPresent otherwise; cannot be updated"`
 	// Optional: Capabilities for container.
```

Comment:
Seems like there should be a note that this and privileged at this level are deprecated and to prefer the ones on security context.